spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'User Show', type: :feature do
+  let(:user) { User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png', bio: 'He is a good programmer') }
+  let!(:post1) { Post.create(author: user, title: 'first post', text: 'first text') }
+  let!(:post2) { Post.create(author: user, title: 'second post', text: 'second text') }
+
+  scenario 'visiting the user Show page' do
+    visit user_path(user)
+
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Bio')
+    expect(page).to have_content('He is a good programmer')
+    expect(page).to have_link('See all posts')
+    expect(page).to have_content('first post')
+    expect(page).to have_link('first post', href: user_post_path(user, post1))
+    expect(page).to have_content('second post')
+    expect(page).to have_link('second post', href: user_post_path(user, post2))
+    expect(page).to have_content('Recent Posts') if page.has_content?('Recent Posts')
```

Comment:
- [ ] Also here, As stated in the project requirement, Please add a test for `asserts you can see the user's profile picture.`, `Tests that asserts you can see the user's first 3 posts.`, `redirects to the post's show page when you click on a user's post.` and `redirects to the user's post's index page when you click on a see all posts`